README.md

```diff
@@ -19,3 +19,21 @@ Documents related to Code for Poznan organization.
 ## License
 
 Documents are licensed under [CC-BY-SA-4.0-PL](https://creativecommons.org/licenses/by-sa/4.0/deed.pl) (with the exception of Code for Poznan logo).
+
+## Welcome message - mail and messenger
+
+Cześć *Tu wpisz imię* !
+
+Kwarantanna nadal trwa i nie wiadomo kiedy wszystko wróci do normy, dlatego kontynuujemy nasze spotkania w wersji online.
+Jeśli chciałbyś/łabyś do nas dołączyć, zapraszamy na spotkanie organizacyjne, wprowadzające dla nowych uczestników, które odbędzie się w środę *tu wpisz dokładną datę* o 18:00 tutaj: [https://meet.google.com/pjv-yerm-cxn](https://meet.google.com/pjv-yerm-cxn)
```

Comment:
Aby uniknąć tej niefortunnej łabyś 👯 

Może:
* Jeśli chcesz do nas dołączyć, zapraszamy na spotkanie, wprowadzające dla nowych uczestników, które odbędzie

Reply:
Jeśli wszyscy są za zamianami to mogę zmienić je na te, które proponuje Jan.

Reply:
dla mnie ok!

Reply:
IMO tak brzmi lepiej `(...) zapraszamy na spotkanie organizacyjne, wprowadzające nowych uczestników, które (...)`